.github/workflows/ci.yml

```
@@ -16,72 +16,72 @@ jobs:
       # Default is true, cancels jobs for other platforms in the matrix if one fails
       fail-fast: false
       matrix:
-        os: [ ubuntu-latest, macos-latest, windows-latest ]
-        go-version: [ 1.22.x ]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        go-version: [1.23.x]
 
         # Set some variables per OS, usable via ${{ matrix.VAR }}
         # XK6_BIN_PATH: the path to the compiled k6 binary, for artifact publishing
         # SUCCESS: the typical value for $? per OS (Windows/pwsh returns 'True')
         include:
-        - os: ubuntu-latest
-          XK6_BIN_PATH: ./cmd/xk6/xk6
-          SUCCESS: 0
+          - os: ubuntu-latest
+            XK6_BIN_PATH: ./cmd/xk6/xk6
+            SUCCESS: 0
 
-        - os: macos-latest
-          XK6_BIN_PATH: ./cmd/xk6/xk6
-          SUCCESS: 0
+          - os: macos-latest
+            XK6_BIN_PATH: ./cmd/xk6/xk6
+            SUCCESS: 0
 
-        - os: windows-latest
-          XK6_BIN_PATH: ./cmd/xk6/xk6.exe
-          SUCCESS: 'True'
+          - os: windows-latest
+            XK6_BIN_PATH: ./cmd/xk6/xk6.exe
+            SUCCESS: "True"
 
     runs-on: ${{ matrix.os }}
 
     steps:
-    - name: Install Go
-      uses: actions/setup-go@v3
-      with:
-        go-version: ${{ matrix.go-version }}
+      - name: Install Go
+        uses: actions/setup-go@v3
```

For consistency across jobs
```suggestion
        uses: actions/setup-go@v5
```